src/version.json

```diff
@@ -1,10 +1,18 @@
 {
     "cli": {
-        "stable": [ "0.5.2", "0.6.2", "0.7.0"],
+        "stable": [ "0.5.2", "0.6.2", "0.7.0", "0.7.1"],
         "prerelease": [ ]
     }, 
     "bridge": {
         "stable": [ "0.5.2", "0.6.2", "0.7.0"],
```

Comment:
Can you please add 0.7.1 as stable bridge version:

 "stable": [ "0.5.2", "0.6.2", "0.7.0", "0.7.1"],